content/en/process/dev-hiring-process.md

```diff
@@ -28,60 +31,79 @@ The interview consists of three stages:
 
 ### Resume Review
 
-Before we even commit to a phone screen , we should review the incoming resume to make sure it’s worth the time on both sides of the table. While we’re not excluding based on exact technologies / experience, it is pretty safe to say that e.g. someone who’s spent the last 10 years on-site at a large corporation mechanically pulling tickets from a queue may not be a great fit for what we’re currently seeking.
+Before we even commit to a phone screen, we should review the incoming resume to make sure it’s worth the time on both
+sides of the table. While we’re not excluding based on exact technologies / experience, it is pretty safe to say that
+e.g. someone who’s spent the last 10 years on-site at a large corporation mechanically pulling tickets from a queue may
+not be a great fit for what we’re currently seeking.
 
 As such, we should go over the following criteria when reviewing resumes (usual caveat about exceptions):
 
-- +/- 3 hours EST. The further from EST, the stronger the rest of the attributes need to be.
-- Has worked remotely before
-- Has worked for a small (< 15 developers, < 50 employee) company
-- Has worked with specific technology in our stack -- Scala, AWS, Big Data, etc
-- Good written communicator, initial communication is clear and understandable
-- (Bonus): initial cover email is not a s/$COMPANY/Narrative template. Did this person even read the job req?
+- +/- 3 hours EST. The further from EST, the stronger the rest of the attributes need to be.
+- Has worked remotely before.
+- Has worked for a small (< 15 developers, < 50 employee) company.
+- Has worked with specific technology in our stack: Scala, AWS, "Big Data", etc.
```

Comment:
Spark (better be specific :))

Reply:
Good call. Fixed in bbc8002e2937c757375680c2a1d81c4ff5ffcbc7